docs/devices.rst

```diff
@@ -60,6 +60,42 @@ type of device. The available iterators are :py:class:`DeviceTokenList` and :py:
    :members: DeviceTokenList, APIDList, DeviceInfo
    :noindex:
 
+
+Open Channel Registration
+--------------
+
+Open Channels are registered by using :py:class:`OpenChannel`.
+
+.. code-block:: python
+
+   import urbanairship as ua
+   airship = ua.Airship(app_key, app_secret)
+
+   my_channel = ua.OpenChannel()
+   my_channel.address = 'my_email@example.com'
+   my_channel.open_platform = 'email'
+   my_channel.opt_in = True
+   my_channel.create(airship)
```

Comment:
I suspect I'm doing something wrong, but I'm getting a 404 response here if I follow the example and it's a new channel (if one was already created with the same address, I get a 200).